LibGUI/GScrollBar.cpp

```diff
@@ -172,7 +172,7 @@ int GScrollBar::scrubber_size() const
 
 Rect GScrollBar::scrubber_rect() const
 {
-    if (!has_scrubber())
+    if (!has_scrubber() || (orientation() == Orientation::Vertical ? height() <= (button_size() * 2) + scrubber_size() : width() <= (button_size() * 2) + scrubber_size()))
```

Comment:
It gets annoying and repetitive that we always have to do things like this:
```cpp
orientation() == Orientation::Vertical ? height() : width()
```

Let's avoid this mess by adding something like this to `GWidget`:

```cpp
int length(Orientation orientation) const { return orientation == Orientation::Vertical ? height() : width(); }
```

Then you can simply use `length(orientation())` everywhere in this code :)